### ISSUE TITLE
Confilct in IE?

### ISSUE THREAD

**Issue by jurgen-74** [blasten/turn.js#297]
Hi everybody.

I implemented turn.js on one of my pages and it works fine in all browser but IE.
Maybe it's because my IE9 is actually working in a virtual machine but I'm pretty sure it's more due a conflict. 
Here is my test-page:
http://tinyurl.com/a84pqe4
All the possible plugins are there. If I disable them (only enable the necessary) it works fine in IE.

Hopefully one of you can help me with this one.

Thanks,
Jurgen

**Comment by jurgen-74** [comment]
I found the problem.
I loaded jQuery 1.5 and this should be 1.7.

The new problem is that when I load 1.7, my printer.preview.js does not work in IE due to a conflict with google custom search... 
Sigh... Sometimes I wonder what all the people, who develop IE, are doing. All other browsers work just fine.

So this is my solution for loading two versions of jQuery:
Add this after every different jQuery load:
script type="text/javascript">
var jQuery_1_5 = $.noConflict(true);
</script
Where the _1_5 needs to address the version of jQuery.
